addons/html_editor/static/src/others/dynamic_placeholder/dynamic_placeholder.js

```diff
@@ -0,0 +1,92 @@
+import { _t } from "@web/core/l10n/translation";
+import { Plugin } from "@html_editor/plugin";
+
+export class DynamicPlaceholderPlugin extends Plugin {
+    static name = "dynamicplaceholder";
+    static dependencies = ["selection", "history", "dom"];
```

Comment:
Is it supposed to work without the qweb plugin? If not, add it into the dependencies.

Reply:
:ok_hand: